integreat_cms/cms/views/utils/hix.py

```diff
@@ -30,9 +31,13 @@ def lookup_hix_score(text):
     :return: The score for the given text
     :rtype: float
     """
-    return TextlabClient(
-        settings.TEXTLAB_API_USERNAME, settings.TEXTLAB_API_KEY
-    ).benchmark(text)
+    try:
+        return TextlabClient(
+            settings.TEXTLAB_API_USERNAME, settings.TEXTLAB_API_KEY
+        ).benchmark(text)
+    except (URLError, OSError) as e:
```

Comment:
Maybe add the same here:
https://github.com/digitalfabrik/integreat-cms/blob/a0326507f663deec5935d5a5d4a291373f5ed996/integreat_cms/textlab_api/apps.py#L38